app/javascript/packs/validateFileInputs.js

```diff
@@ -36,57 +38,39 @@ const PERMITTED_FILE_TYPES = ['image'];
  */
 const MAX_FILE_NAME_LENGTH = 250;
 
-/**
- * Removes any pre-existing error messages from the DOM related to file
- * validation.
- *
- * @param {HTMLElement} fileInput - An input form field with type of file
- */
-function removeErrorMessage(fileInput) {
-  const errorMessage = fileInput.parentNode.querySelector(
-    'div.file-upload-error',
-  );
-
-  if (errorMessage) {
-    errorMessage.remove();
-  }
-}
-
 /**
  * Adds error messages in the form of a div with red text.
  *
- * @param {HTMLElement} fileInput - An input form field with type of file
  * @param {string} msg - The error message to be displayed to the user
  *
  * @returns {HTMLElement} The error element that was added to the DOM
  */
-function addErrorMessage(fileInput, msg) {
-  const fileInputField = fileInput;
-  const error = document.createElement('div');
-  error.style.color = 'red';
-  error.innerHTML = msg;
-  error.classList.add('file-upload-error');
-
-  fileInputField.parentNode.append(error);
+function addErrorMessage(msg) {
+  if (top.addSnackbarItem) {
+    // The Comment editor's context (MarkdownToolbar component) doesn't have
+    // access to the Snackbar element in the DOM, so it needs to use `top`
+    top.addSnackbarItem({
+      message: msg,
+      addCloseButton: true,
+    });
+  } else {
+    // The Post editor (Toolbar component) doesn't have access to
+    // `top.addSnackbarItem` so we need to check to ensure if it's undefined
+    addSnackbarItem({
+      message: msg,
+      addCloseButton: true,
+    });
+  }
 }
 
 /**
  * Handles errors for files that are too large.
  *
  * @param {object} fileSizeErrorHandler - A custom function to be ran after the default error handling
- * @param {HTMLElement} fileInput - An input form field with type of file
  * @param {number} fileSizeMb - The size of the file in MB
  * @param {?number} maxFileSizeMb - The max file size limit in MB
  */
-function handleFileSizeError(
-  fileSizeErrorHandler,
-  fileInput,
```

Comment:
`fileInput` element was passed into these error message callback functions so the `<div>` containing the error would be placed correctly, which is no longer needed since the message is being displayed via the Snackbar. A few occurrences of this throughout the PR